test/rubygems/test_gem_ext_cargo_builder.rb

```diff
@@ -57,15 +55,35 @@ def test_build_cdylib
     end
 
     output = output.join "\n"
-
     bundle = File.join(@dest_path, "release/rust_ruby_example.#{RbConfig::CONFIG['DLEXT']}")
 
-    require(bundle)
+    assert_match "Finished release [optimized] target(s)", output
+    assert_ffi_handle bundle, 'Init_rust_ruby_example'
+  rescue Exception => e
+    pp output if output
```

Comment:
Is this intended to be left in here?

Reply:
Yes, it really does make all of the difference when debugging. I can remove it if you prefer though!

Reply:
Looks strange to me too, but I can see how it's useful, and the code was already there, so ok!